integrations/serverpod_cloud_storage_gcp/lib/src/aws_s3_upload/aws_s3_upload.dart

```diff
@@ -61,9 +61,13 @@ class GCPS3Uploader {
     try {
       final res = await req.send();
 
+      if (res.statusCode >= 400 && res.statusCode < 500) {
+        print('Failed to upload to GCP, with reason: ${res.reasonPhrase}');
+      }
+
       if (res.statusCode == 204) return '$endpoint/$uploadDst';
     } catch (e) {
-      print('Failed to upload to AWS, with exception:');
+      print('Failed to upload to GCP, with exception:');
```

Comment:
Suggestion: Could we rename these files to match what they are.

This is quite confusing right now. Or are we using some aws interface for GCP?

Reply:
We are using the aws api that Google has support for. E.g. they have implemented the same interface to make transitions from s3 easier.

Reply:
Aah ok! 👍